svm-transaction/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+mod instruction;
+mod message_address_table_lookup;
+mod svm_message;
+mod svm_transaction;
+
+pub use {instruction::*, message_address_table_lookup::*, svm_message::*, svm_transaction::*};
```

Comment:
Just my own bias here - you can take it or leave it - but I tend to avoid glob exports like these, and instead make the modules public and only mark those items within each module you want to export as public.

Reply:
removed glob exports here: [38e35bc](https://github.com/anza-xyz/agave/pull/1918/commits/38e35bcc33c93b6835e09efd550e288827f02da1)